cmd/pullrequest.go

```diff
@@ -43,4 +47,24 @@ var pullrequestCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(pullrequestCmd)
+	
+	pullrequestCmd.Flags().StringVarP(
+		&LinkType,
+		"link-type",
+		"l",
+		"close",
+		"The type of link to create with the associated issue.",
+	)
+}
+
+func validateLinkType() {
```

Comment:
This should be in a util class probably rather than in one of the "plugin" modules

Reply:
Moved into util class.